src/v/cluster/types.h

```diff
@@ -514,7 +514,14 @@ class configuration_invariants_changed final : public std::exception {
 };
 // delta propagated to backend
 struct topic_table_delta {
-    enum class op_type { add, del, update, update_finished, update_properties };
+    enum class op_type {
+        add,
+        add_materialized,
```

Comment:
I didn't find any cases of this being serialized, but in general it's safer to add a new one to the end.